addons/sosivio/enable

```diff
@@ -2,58 +2,191 @@
+            --set expose=LoadBalancer --set adminPassword=microsivio --version=$HELM_VERSION"
+  print_debug "applying helm chart"
+  $HELM_CMD sosivio $HELM_REPO_NAME/sosivio &> /dev/null
+
+}
+
+wait_for_sosivio () {
+
+  echo ""
+  echo "Waiting for Sosivio to become active..."
+  echo "It might take a while if your network connection is slow."
+
+  WATCH="$KUBECTL rollout status --watch"
+  start_time=$(date +'%s')
+  until $WATCH deploy -n ${NAMESPACE_SOSIVIO} sosivio-dashboard --watch > /dev/null 2>&1
+  do
+    if [ $(( $start_time + $WAIT_SECONDS )) -lt $(date +'%s') ] ; 
+    then
+        echo "sosivio wait time exceeded ($WAIT_SECONDS seconds)"
+        echo "please check your internet connection or run again with '-w' flag."
+        echo "check 'microk8s enable sosivio -h' for more details"
+        echo ""
+        echo "note: sosivio is still getting deployed, you can check if all the pods are running using:"
+        echo "microk8s kubectl get pods -n $NAMESPACE_SOSIVIO"
+        exit 1
+    fi
+    sleep 1
+  done
+
+}
+
+
+print_finished () {
+  echo ""
+  echo ""
+  echo "Sosivio is reachable using the following local address:"
+  echo -e "${BOLD}http://localhost:$(${KUBECTL} get svc ${DASHBOARD_SERVICE} -n ${NAMESPACE_SOSIVIO} -o jsonpath='{.spec.ports[0].nodePort}')${NC}"
+  echo ""
+  echo "Alternatively you can use port-forward:"
+  echo "'microk8s kubectl port-forward -n ${NAMESPACE_SOSIVIO} svc/${DASHBOARD_SERVICE} 8088'"
+  echo ""
+  echo "Your first login credentials are:"
+  echo "Username: admin"
+  echo "Password: $(${KUBECTL} get secret -n ${NAMESPACE_SOSIVIO} sosivio-admin-otp -o jsonpath='{.data.password}' | base64 -d)"
+  echo ""
+  echo "You can get this password again using the following command:"
+  echo "microk8s kubectl get secret -n ${NAMESPACE_SOSIVIO} sosivio-admin-otp -o jsonpath='{.data.password}' | base64 -d"
+  echo ""
+  echo -e "${BLUE}Sosivio Enabled${NC}"
+}
+
+
+
+################# main #################
+
+
+while getopts "dv:u:n:h" opt; do
+  case $opt in
+    d)
+      DEBUG=1
+      ;;
+    v)
+      if [ -z $OPTARG ] ; then
+        opt_error "error: -v flag requires parameter."
+      fi
+      HELM_VERSION=$OPTARG
+      ;;
+    u)
+      if [ -z $OPTARG ] ; then
+        opt_error "error: -u flag requires parameter."
+      fi
+      HELM_REPO_URL=$OPTARG
+      ;;
+    n)
+      if [ -z $OPTARG ] ; then
+        opt_error "error: -n flag requires parameter."
+      fi
+      HELM_REPO_NAME=$OPTARG
+      ;;
+    *)
+      opt_error "Invalid option: -$OPTARG"
+      ;;
+  esac
 done
+echo "Sosivio Version $HELM_VERSION"
+echo "hint: use -v flag to set a different version. (ex - microk8s enable sosivio -v x.y.z)"
+sleep 3
+
+if [ $DEBUG -eq 1 ]; then 
+  echo "debug flag set" >&2
+  echo "repo url is set to $HELM_REPO_URL"
+  echo "repo name is set to $HELM_REPO_NAME"
+fi
+
+install_addons_prereq
+install_soivio
+wait_for_sosivio
+print_finished
```

Comment:
I think we forgot to call the `add_helm_repo` function.

```suggestion
install_addons_prereq
add_helm_repo
install_soivio
wait_for_sosivio
print_finished
```

Reply:
We should probably update respective version PRs #175 and #176 with this change as well.

Reply:
Any update on this @DanArlowski ?